### ISSUE TITLE
create a circular heterogeneity with inparam_hetero ?

### ISSUE THREAD

**Issue by Hecti90** [geodynamics/axisem#60]
I would like to do axisem simulations with a circular heterogeneity (Vp,Vs and rho) below the surface and some distance from the sorurce.
Can this be done with the inparam_hetero file? The documentation is currently limited to
some inline comments which I am not sure to fully understand.

**Comment by martinvandriel** [comment]
Yes, that is the correct way to do it. It's not very well tested and only used by few people, so I guess there is no way around digging in a bit.
